src/tabs/auth.tsx

```diff
@@ -51,34 +37,28 @@ export function AuthApp({ initialScreenType }: AuthAppProps) {
       />
     );
   } else if (initialScreenType === "default") {
-    content = (
-      <Router hook={useAuthRequestsLocation}>
-        <Route path="/connect" component={Connect} />
-        <Route path="/allowance" component={Allowance} />
-        <Route path="/token" component={Token} />
-        <Route path="/sign" component={Sign} />
-        <Route path="/signKeystone" component={SignKeystone} />
-        <Route path="/signature" component={Signature} />
-        <Route path="/subscription" component={Subscription} />
-        <Route path="/signDataItem" component={SignDataItem} />
-        <Route path="/batchSignDataItem" component={BatchSignDataItem} />
-      </Router>
-    );
+    content = <Routes routes={AUTH_ROUTES} diffLocation />;
```

Comment:
Routes have been defined in `AUTH_ROUTES` and are wrapped automatically in a `Page` component by the `Routes` component.

Additionally, the `HeadAuth` component could be extracted here so that it is not affected by page transitions, but that would also require adding a `headProps` property to the `RouterConfig` interface (for static personalization) and also creating a context and/or portal to allow for personalization coming directly from the view/page's own life cycle (for dynamic personalization).